### ISSUE TITLE
Random GPU's coming up with 0 Sol/s

### ISSUE THREAD

**Issue by Meatball13** [BeamMW/cuda-miner#7]
Backstory.  Have multiple 12 NVidia GPU rigs.  Everytime I launch the miner 8-10 of the GPU's start up fine, but 2-4 of them just sit a 0 Sol/s and never hash.  Additionally, the 2-4 that won't hash will vary after every launch, so I don't believe it's a physical hardware issue.

I found that if I use the --devices switch and put all but 1 card in there (for 12 GPU it would be --devices 0,1,2,3,4,5,6,7,8,9,10) all 11 cards work fine.  As soon as I either add the last card (put ,11 on the end of devices) or remove the --devices flag to use all GPU's, I fall back to 2-4 GPU's not hashing.  As long as you call out all the cards but 1 in your --devices, all of those will work.

Not a coder, but maybe there's a function or process that isn't closing gracefully when it hits that last GPU?

Update: Think I was way off base here.  But I think I may have figured something out.  I think it's all Virtual Memory related.  Most of my machines had 48 GB of VM and 4-12 GB physical RAM.  Bumping my VM up to 64 GB, I was able to run all 12 without a problem.  Must be a big hit to VM during the card Initialization and if VM runs out, some cards probably never initialize.  

Long story short, I don't think this is a direct Bug, but might want to put out a warning for folks to use more VM.
